blackbox/docs/configuration/index.txt

```diff
@@ -46,8 +43,12 @@ Settings are applied in the following order:
 Each setting value overwrites any previous value. So, for example, command line
 settings will override configuration file settings.
 
-Topics
-======
+.. NOTE::
+
+   If you're just getting started with a particular part of CrateDB, we
+   recommend you consult the appropriate top-level section of the
```

Comment:
recommend you to consult ...

Reply:
hmm. I don't think so. sounds wrong to my ear! thx though!

Reply:
@nslater i guess you are right ;)